src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import dao.FileReaderImpl;
+import dao.FileWriter;
+import dao.FileWriterImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import model.OperationType;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import service.FruitShopService;
+import strategy.BalanceOperationHandler;
+import strategy.OperationHandler;
+import strategy.PurchaseOperationHandler;
+import strategy.ReturnOperationHandler;
+import strategy.SupplyOperationHandler;
+
+public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        if (args.length < 2) {
```

Comment:
It's good practice to extract magic numbers into constants. The number of expected arguments '2' can be named something like `EXPECTED_ARGUMENTS_COUNT` to make the code more readable and maintainable.